### ISSUE TITLE
Full-time Clarity Educator

### ISSUE THREAD

**Issue by cuevasm** [stacksgov/residence-program#14]
In the past few months, demand for Clarity developer talent has spiked across the ecosystem. There are open positions and most companies/projects in the ecosystem find themselves in a constant state of seeking out those with Clarity skills for feedback and support. Efforts like Clarity Universe have started to help alleviate this talent shortage, but between supporting accelerator teams, working on key blockchain improvements, the grants program, and running Clarity Universe itself, we've hit the bandwidth limit. 

Chatting with some other projects in the space, there's a willingness and desire to hold more Clarity Universe (or similar flavored versions) much more often and to support specific efforts. With that, I'm proposing a full-time Clarity Educator role. Please use this issue to signal your support for this position and provide feedback on the exact mix of duties. 

TL;DR: We need more people that know Clarity. A full-time Clarity educator, leveraging the content we've already created, can rapidly infuse the ecosystem with talent as well as push broader education around smart contracts for Bitcoin forward.

-----

**Proposed Title of Position**
Clarity Education Lead

**Focus Areas/Scope of Work**
You will be the go-to starting point for all things related to learning the Clarity smart contract language and play a critical role in growing the use of the language globally.

**Responsibilities**
- Create, source, and organize various educational materials, live trainings, events/cohorts and mini projects to extend Clarity Universe and variations throughout the ecosystem
- Support Stacks Chapters, universities, and other entities in running Clarity education programming as needed
- Analyze quantitative and qualitative metrics to evaluate the effectiveness of a Clarity Education program - and make adjustments as necessary
- Scale Clarity education programs by utilizing resources at your disposal, bringing on collaborators and documenting best practices for others to independently run similar programs
- Support the ecosystem in creating and sharing best practices for the language
- On occasion, directly support builders with feedback and advice on how their projects
- Support new developers at ecosystem events, hackathons, etc. with Clarity material and workshops

**Requirements** (or similar):
- Expertise and demonstrated proficiency in Clarity and at least one other smart contract language
- 4+ years of program or project management experience, preferably in the tech industry
- 4+ years experience in a teaching position
- Comfortable and polished presenter, in-person and virtually

**Timeline**
I see this position as being needed **immediately**. There are numerous opportunities to take Clarity into some pretty exciting channels including household name universities and well-known online learning platforms. This is a high-leverage position I'd love to start as soon as possible!

**Interested? Apply here:** https://stacks.org/stacks-resident

**Comment by pstan26** [comment]
The City of Miami has signaled an intent to expand critical educational functions run by the city (think Stacks and MIA crypto school/university) and that this role would greatly help them accelerate that by supporting them upfront. Can't plus one this role enough. Would go an incredibly long way for Stacks, MiamiCoin, and Miami.

**Comment by jennymith** [comment]
Hi all, if you're interested in this role, please apply here: [Hi all, if you're interested in this role, please apply here: https://stacks.org/stacks-resident. I've also updated the role description with the link.](https://stacks.org/stacks-resident). I've also updated the role description with the link.

**Comment by cuevasm** [comment]
Moving forward with applications. 2500 STX referral bonus to the person that refers the candidate chosen. 🚀

**Comment by migustamim** [comment]
I recommend @empeje for this position.

**Comment by JakeBlockchain** [comment]
Any update on finding someone for this?
Asking devs around the ecosystem, I'm also seeing a need for someone who understands the nuances of building on Clarity, vs Rust on Solana vs Solidity on Ethereum.
Would be a good way to combine what other devs are familiar with on other systems to the comparable on Stacks.

It's something I'd be interested in deep diving into.
For example, The coding basics of launching a 100 piece NFT project on Stacks, Solana, and Ethereum and making a series of videos and write-ups about the differences/similarities of each.

I'm a new dev though, so i'd only gain the knowledge to be something like a Clarity educator overtime as I use the different tools and languages.

**Comment by cargocultscience** [comment]
I'm a fairly long time coder coming from c/c++/python, currently learning Clarity.  One thought I've had is to create material to show clarity vs more well known languages (eg python and typescript) as a means to bridge over coders that want to pick up Clarity.  As a Clarity newbie myself I may be in a good position to create this.  If this is interesting to anyone please reach out.

**Comment by cuevasm** [comment]
Hey all, quick update here that we received a number of private applications through the secondary application option linked above and have screened/interviewed all of them. A top candidate has emerged and in the next few days they'll submit their information as a new issue and proceed through the committee and community presentation steps. So excited about this one, absolute top notch team with great reccs and track record for Web 3 developer education. 

In addition, we're also in talks with a few other teams and individuals to play further supporting roles around Clarity education/content to complement the work of the Clarity Eductator. Within a month, I expect to have a robust group focused around various aspects of Clarity Universe, standby for details on when these residents will be presenting to community, should be in the next couple of weeks! 🚀
